src/sentry/integrations/github/integration.py

```diff
@@ -33,14 +33,6 @@
 """
 
 FEATURES = [
-    FeatureDescription(
```

Comment:
Note that I re-ordered some of the project management + source code management integrations so the two tags show up in a consistent order.